schema/release-schema.json

```diff
@@ -1405,7 +1405,7 @@
         },
         "classification": {
           "title": "Classification",
-          "description": "The primary classification for the item.",
+          "description": "The primary classification for the item. Items should be classified using existing item classification schemes, such as the [EC Common Procurement Vocabulary (CPV)](https://simap.ted.europa.eu/web/simap/cpv).",
```

Comment:
I think this sentence was unnecessary in the original documentation, and we can just remove it. Any scheme used by publishers exists (it's not like they invented one just for OCDS).